dyn_em/module_big_step_utilities_em.F

```diff
@@ -6684,4 +6684,51 @@ END SUBROUTINE conv_t_tendf_to_moist
 
 !==============================================================================
 
+  subroutine Cloud_tracer_nudge (dt, dt_relax, dt_nudge, xtime, qc, qi, qs,  &
+      tr_qc, tr_qi, tr_qs, ids, ide, jds, jde, kds, kde, ims, ime, jms, jme, &
+      kms, kme, its, ite, jts, jte, kts, kte)
+
+    implicit none
+
+    real, intent(in) :: dt, dt_relax, dt_nudge, xtime
+    real, dimension (ims:ime, kms:kme, jms:jme), intent (inout) :: qc, qi, qs
```

Comment:
@pedro-jm 
What protections are in place to protect against using this scheme if Qc, Qi, Qs do not exist with the selected MP scheme.

Reply:
I see the test in phy_init